code/modules/chemistry/Reagents-Drugs.dm

```diff
@@ -472,13 +474,13 @@ datum
 
 			on_add()
 				if(istype(holder) && istype(holder.my_atom) && hascall(holder.my_atom,"add_stam_mod_regen"))
-					remove_buff = holder.my_atom:add_stam_mod_regen("consumable_good", 1)
+					remove_buff = holder.my_atom:add_stam_mod_regen("r_nicotine", 1)
```

Comment:
Any reason you didn't do the `if (ismob(holder?.my_atom))` etc. stuff here in addition? There's a few others you didn't touch either. I think we should be consistent with how we check/apply them, at least.